package.json

```diff
@@ -2,31 +2,16 @@
   "name": "yargs",
   "version": "17.7.2",
   "description": "yargs the modern, pirate-themed, successor to optimist.",
-  "main": "./index.cjs",
+  "main": "./index.mjs",
   "exports": {
     "./package.json": "./package.json",
-    ".": [
-      {
-        "import": "./index.mjs",
-        "require": "./index.cjs"
-      },
-      "./index.cjs"
-    ],
     "./helpers": {
-      "import": "./helpers/helpers.mjs",
-      "require": "./helpers/index.js"
+      "import": "./helpers/helpers.mjs"
```

Comment:
Not sure if the lone import was intentional. I was expecting just file path rather than conditional.

```
".helpers": "./helpers/helpers.mjs"
```

Reply:
Done.